src/batch-client/index.ts

```diff
@@ -525,6 +526,22 @@ export class ZimbraBatchClient {
 			}
 		});
 
+	public setRecoveryAccount = ({
+		op,
+		recoveryAccount,
+		recoveryAccountVerificationCode,
+		channel
+	}: SetRecoveryAccountOptions) =>
```

Comment:
Since this entire object is passed into `jsonRequest`, we can shorten this to:

```js
	public setRecoveryAccount = (options: SetRecoveryAccountOptions) =>
		... snip ...
			body: options
```